### DISCUSSION TITLE
Monitoring MM2 with JMX exporter?

### DISCUSSION THREAD

**Discussion by iftachbyaf** [strimzi/strimzi-kafka-operator#10131] [Q&A] [answered]
Hey all,

We want to monitor our MM2 deployment installed via Strimzi. Added MetricsConfig and the configmap as per
https://github.com/strimzi/strimzi-kafka-operator/blob/main/examples/metrics/kafka-mirror-maker-2-metrics.yaml

Im not sure I understand if its working as expected now?

Is the JMX exporter supposed to run on the same pod as the mirrormaker deployment? how do i verify its working properly/access the metrics now? (assuming its supposed to write them to some endpoint which I can scrape? not sure which though)

I can see on my mirroring pods `KAFKA_CONNECT_METRICS_ENABLED: true` , but otherwise not sure how to proceed... 
Any tips would be appreciated :)

**Comment by scholzj** [comment, marked as answer]
You can check if the metrics are exposed in the pod on the port 9404 on the `/metrics` path. They are exposed directly from the Mirror Maker 2 container.

**Comment by iftachbyaf** [comment]
Cool. That seems to be working as expected. Thanks for the quick reply!
